openbb_terminal/terminal_controller.py

```diff
@@ -571,8 +589,19 @@ def call_exe(self, other_args: List[str]):
                 )
                 time.sleep(3)
             elif ns_parser.file:
+                # if string is not in this format eg)"default/file.openbb" then check for files in ROUTINE_FILES
+                # regex is used to match strings that start with C:/ and remove it so there won't be duplicates
+                # for user who starts the string with Users/username
                 file_path = " ".join(ns_parser.file)
-                path = self.ROUTINE_FILES.get(file_path, Path(file_path))
+                full_path = Path('C:/' + re.sub(r'^(C:\/?|c:\/?)', '', file_path).strip())
```

Comment:
I am not sure why it is not working with the regex for you as it is the opposite for me and 
exe --file /Users/username/OpenBBUserData/routines/hub/personal/The Influence of the Central Bank.openbb only works for me with the regex otherwise i need to include the C drive at the start for it to work without regex.

> I've made a couple suggestions and fixed linting, other than that looks good to me!

Thank you!

Reply:
But I guess it is better to remove the regex as the full path with the C:/ at the start still works with or without it. The purpose of the regex was for users to be able to choose to include the C;/  or not include it and the path will still work but at the end of the day the issue seems trivial and it was simply for convenience